### PR TITLE
updated code and image for new organiser Tapati Das

### PR DESCRIPTION
**Pull request by tapatidas123**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -116,8 +116,16 @@
 							<figcaption>Nikita Gandhi</figcaption>
 						</figure>
 					</a>
-					</li>	
-					
+					</li>
+					<li>
+					<a href="https://in.linkedin.com/in/tapati-das-742704117" target="_blank">
+					        <figure>
+						        <img class"organiser_img" src="images/tapati.jpg"/>
```
Comment (PyLadiesPune) [inline index.html]: You missed an equal to sign after class. Due to that the size parameters from CSS code don't apply to the image and it's displaying as a large size image.

**Comment by PyLadiesPune** [comment]
Hi Tapati. You need to make a small correction in the code. I've left an inline comment for the same. Always run your HTML code on the browser to check if it's working as expected, before sending in a PR.
Also, can you please use a more descriptive message for your commits and PR. Maybe something like "Updated html and added image for new organiser Tapati Das"?
Thanks. :)

**Comment by PyLadiesPune** [comment]
Thanks for making the change. Hate to be picky but there's a black border on the circumference of your image. Since other images do not have a boundary can you just edit the border out? Apart from that everything all looks good!

**Comment by Shrreya** [comment]
Thanks for correcting the border. But once again your commit messages should be descriptive of the action you've performed. It would really be helpful if your last commit was something like "Removed border from image". You may edit your commit message by running "git commit --amend", editing the message, then running your git push command with the --force flag. Try it?

**Pull request merged**